shm/Ship.hpp

```diff
@@ -9,27 +9,21 @@
 class Ship {
 public:
     Ship()
-        : id_(0)
-    {}
+        : id_(0) {}
     Ship(int capacity, int maxCrew, int speed, const std::string& name, size_t id)
-        : capacity_(capacity)
-        , maxCrew_(maxCrew)
-        , crew_(0)
-        , speed_(speed)
-        , name_(name)
-        , id_(id)
-    {}
+        : capacity_(capacity), maxCrew_(maxCrew), crew_(0), speed_(speed), name_(name), id_(id) {}
     Ship(int maxCrew, int speed, size_t id)
-        : Ship(0, maxCrew, speed, "", id)
-    {}
+        : Ship(0, maxCrew, speed, "", id) {}
 
     void setName(const std::string& name) { name_ = name; }
 
-    size_t getCapacity() const  { return capacity_; }
-    size_t getMaxCrew() const   { return maxCrew_; }
-    size_t getSpeed() const     { return speed_; }
+    size_t getCapacity() const { return capacity_; }
+    size_t getMaxCrew() const { return maxCrew_; }
+    size_t getSpeed() const { return speed_; }
     std::string getName() const { return name_; }
-    size_t getId() const        { return id_; }
+    size_t getId() const { return id_; }
```

Comment:
Btw. Why did you change those lines? Is it related to comment in PR with our OOP_homework1 ? About keeping one format?

Reply:
It wasn't me, it was my auto format options. I wrote about it just after I pushed changes and noticed that.